### PR TITLE
fix(frontend): remove the asterisks indicating mandatory fields.

### PR DESCRIPTION
**Pull request by denisquesada**
## Description

* Removed the asterisks indicating mandatory fields in admin pages.
* Added an aria label to the **Edit** button on the `/admin/settings/topicarea` page.
* Updated the aria label to the **Home link** on the `Admin` home screen to match the visual text reads.

## Testing

Describe how you tested your changes and/or give instructions to the reviewers on how they can verify them.

1. Go to https://d2zdyv8kmq5ffn.cloudfront.net/
2. Verify the color changes.

---

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: frontend/src/components/Combobox.tsx [hunk 1]
```diff
@@ -55,7 +55,6 @@ function Combobox(props: Props) {
                 htmlFor={props.id}
             >
                 {props.label}
-                {props.label && props.required && <span>&#42;</span>}
```
Comment (mabreuortega) [inline frontend/src/components/Combobox.tsx:58]: Why are we doing this? I remember we put * at some point

**Review by mabreuortega — COMMENTED** [review]
question added

File: frontend/src/components/Combobox.tsx (hunk 1 above)
Comment (denisquesada) [reply frontend/src/components/Combobox.tsx:58]: Upon agreement we have decided to get it rid of

**Review by mabreuortega — APPROVED** [review]
LGTM

**Pull request merged**